src/Plugin.Settings.Shared/CrossSettings.cs

```diff
@@ -9,6 +9,7 @@
  */
 
 using System;
+using System.Diagnostics;
```

Comment:
Doesn't look like this needs to be checked in...

Reply:
yeah, it slipped between cracks. I will update it :)

Reply:
Fixed this, read for merge